statsmodels/base/tests/test_penalized.py

```diff
@@ -0,0 +1,420 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May 10 12:39:33 2015
+
+Author: Josef Perktold
+License: BSD-3
+"""
+
+import numpy as np
+from numpy.testing import assert_allclose, assert_equal
+from statsmodels.discrete.discrete_model import Poisson, Logit, Probit
+from statsmodels.genmod.generalized_linear_model import GLM
+from statsmodels.genmod.families import family
+from statsmodels.base._penalized import PenalizedMixin
+import statsmodels.base._penalties as smpen
+
+class PoissonPenalized(PenalizedMixin, Poisson):
+    pass
+
+class LogitPenalized(PenalizedMixin, Logit):
+    pass
+
+class ProbitPenalized(PenalizedMixin, Probit):
+    pass
+
+class GLMPenalized(PenalizedMixin, GLM):
+    pass
```

Comment:
`pytest.mark.skip` for these?  Marking unfinished tests explicitly will make it harder for the follow-ups to slip through the cracks.

Reply:
These are not test classes, These are the penalized model classes that work completely by inheritance, i.e. It's one of the main points of this PR.
For now there are no public penalized classes. I'm just developing a generic penalized framework for doing this.

Reply:
My mistake, thanks for clarifying.